packages/core/src/provider/base.ts

```diff
@@ -100,7 +100,7 @@ export class SmartAccountProvider<
 
     this.txMaxRetries = opts.txMaxRetries;
     this.txRetryIntervalMs = opts.txRetryIntervalMs;
-    this.txRetryMulitplier = opts.txRetryMulitplier;
+    this.txRetryMultiplier = opts.txRetryMultiplier;
```

Comment:
oh oof... I don't think we can make this change without it being a breaking change :/ We have plans to make a breaking change when we upgrade to viem 2.0 so we can address this then